src/test/java/fi/aalto/cs/intellij/actions/EnablePluginsNotificationActionTest.java

```diff
@@ -0,0 +1,58 @@
+package fi.aalto.cs.intellij.actions;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.intellij.ide.plugins.IdeaPluginDescriptor;
+import com.intellij.notification.Notification;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.extensions.PluginId;
+import fi.aalto.cs.intellij.PluginsTestHelper;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class EnablePluginsNotificationActionTest extends PluginsTestHelper {
+
+  @Test
+  public void testActionPerformed() {
+    // given
+    List<IdeaPluginDescriptor> descriptorList = getDummyPluginsListOfTwo();
+    Map<PluginId, IdeaPluginDescriptor> descriptors = new HashMap<>();
+
+    descriptorList.forEach(descriptor -> {
+      descriptor.setEnabled(false);
+      descriptors.put(descriptor.getPluginId(), descriptor);
+    });
```

Comment:
I guess it's a matter of taste but I'd avoid loops and such complex code constructions in tests and try to write more "idiot proof" like:
```
descriptor1.setEnabled(false);
descriptor2.setEnabled(false);

descriptors.put(descriptor1);
descriptors.put(descriptor2);
```